map/src/data/markers.ts

```diff
@@ -1070,6 +1084,659 @@ This Mutual Aid Network is a grassroots, community focused and lead ecosystem fo
+Due to so many local closures and loss of income for some now, and many others possibly in the near future, we as usual want to do our part to help out those who are most vulnerable in our community. Please help us with a monetary donation or donation of canned, boxed or bagged food items, baby formula, produce, toiletries (including pampers) and over the counter medical supplies.
+For more info, or to set up a scheduled drop off of items, or to volunteer please message us or email us at: maroonmovement@gmail.com. Thank you for your solidarity!`,
+    services: ['aid', 'medicine', 'food', 'supplies', 'information'],
+    contact: {
+      general: {
+        web: {
+          Website: '',
+          Twitter: '',
+          Instagram: '',
+          PayPal: 'Paypal: fcrcollective@gmail.com',
+          Cashapp: '$Simaleerbg',
+          Venmo: 'Venmo: @Simaleerbg',
+        },
+        email: ['maroonmovement@gmail.com'],
+        phone: [''],
+        facebookGroup:
+          'https://www.facebook.com/donate/201582851152373/563301290955017/',
+      },
+      getHelp: {
+        email: ['maroonmovement@gmail.com'],
+      },
+      volunteers: {
+        email: ['maroonmovement@gmail.com'],
+        web: {
+          Donate:
+            'https://www.facebook.com/donate/201582851152373/563301290955017/',
+        },
+      },
+    },
+    loc: LOCATIONS.USA.MD_BALTIMORE,
+  },
+  {
+    contentTitle: 'Baltimore Neighborhood Quarantine Response Teams',
+    services: ['aid', 'supplies'],
+    contact: {
+      general: {
+        web: {
+          Website:
+            'https://docs.google.com/document/d/1G5JcyxmywcTdNeaSGxUNycl9mtCdGkF4MYPVxEBH4SY/mobilebasic?urp=gmail_link',
+        },
+        email: ['celesteperilla@gmail.com'],
+      },
+      getHelp: {
+        web: {
+          Website:
+            'https://docs.google.com/document/d/1G5JcyxmywcTdNeaSGxUNycl9mtCdGkF4MYPVxEBH4SY/mobilebasic?urp=gmail_link',
+        },
+      },
+      volunteers: {
+        email: ['celesteperilla@gmail.com'],
+        web: {
+          Website:
+            'https://drive.google.com/drive/u/0/folders/1PcTEcMtjdLzak7C_ty6e-5pHM_LHJu9J',
+        },
+      },
+    },
+    loc: LOCATIONS.USA.MD_BALTIMORE,
+  },
+  {
+    contentTitle: 'BENZIE COUNTY COVID-19 RESOURCES AND NEEDS',
+    services: ['aid', 'information'],
+    contact: {
+      general: { web: { Website: '' }, email: [], facebookGroup: '' },
+    },
+    getHelp: { email: [], web: {} },
+    volunteers: {
+      email: [],
+      web: {
+        Volunteer:
+          'https://docs.google.com/document/d/1FYcqcbY2qMuuy75GIIM9UuqvFR_-YlVKfFcrqJOy9mw/edit?fbclid=IwAR1Lfd0mZrm4GgqOwmcXAeJGnTLUEQ0wmooezGvgnVYjXvKEZT8EYPU2tUI',
+      },
+    },
+    contentBody: '',
+    loc: {
```

Comment:
We should pull these locations out as specific locations. It looks like it's smaller towns within Michigan.

Reply:
ok, i scraped these from another data source, ill see if i can get some better location descriptions

Reply:
Ok sounds good. ^

Reply:
i did find an api to convert the latlng to reasonable place names, thanks for looking this over.
please let me know what else i can do. cheers

Reply:
this is pretty excellent honestly.

Reply:
thank you. i think i resolved the conflicts. much appreciated for your patience